src/qibo/backends/numpy.py

```diff
@@ -75,6 +77,12 @@ def to_numpy(self, x):
     def compile(self, func):
         return func
 
+    def _clifford_pre_execution_reshape(self, state):
+        return state
+
+    def _clifford_post_execution_reshape(self, state, nqubits):
+        return state
+
```

Comment:
These as well, there is no need to keep them in `NumpyBackend`

Reply:
Mmmh but then how do you diversify between the different engines (backends)? Because in the `cupy` backend these are linearizing the matrix.

Reply:
We're already discussing the issue in the former comment. I'm closing this to reduce noise.